projectGuide/easyshop-kind.md

```
@@ -361,33 +313,36 @@ kubectl apply -f kubernetes/11-hpa.yaml
-> ### 4. Apply the Configurations
-> ```bash
-> # Apply all configurations
-> kubectl apply -f kubernetes/app-deployment.yaml
-> kubectl apply -f kubernetes/migration-job.yaml
-> kubectl apply -f kubernetes/ingress.yaml
-> 
-> # Verify the deployment
-> kubectl get pods -n easyshop
-> kubectl get ingress -n easyshop
+> ##### 10. Update Migration Job
+> ###### Create or update `kubernetes/12-migration-job.yaml`:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)